ui.tests/test-module/specs/panelcontainer/panelcontainer.runtime.complex.spec.js

```diff
@@ -15,6 +15,115 @@
     let formContainer = null;
+    const selectMap = {
+        "OK": "yes",
+        "I don't think so": "no"
+    };
+    const data = [
+        {
+            "textInput": "Green",
+            "numInput": 70,
+            "panel1": ["OK", "I don't think so", "OK"],
+            "panel2": [
+                {
+                    "date" : "2023-02-24",
+                    "email" : "test@test.com"
+                },
+                {
+                    "date" : "2023-02-25",
+                    "email" : "blue@blue.com"
+                },
+                {
+                    "date" : "2023-02-14",
+                    "email" : "qw@qw.com"
+                }
+            ]
+        },
+        {
+            "textInput": "Orange",
+            "numInput": 98,
+            "panel1": ["I don't think so", "OK", "I don't think so"],
+            "panel2": [
+                {
+                    "date" : "2023-02-28",
+                    "email" : "ty@ty.com"
+                },
+                {
+                    "date" : "2023-03-04",
+                    "email" : "p@p.com"
+                }
+            ]
+        },
+        {
+            "textInput": "Horse",
+            "numInput": 54,
+            "panel1": ["OK", "I don't think so"],
+            "panel2": [
+                {
+                    "date" : "2023-03-11",
+                    "email" : "try@y.com"
+                },
+                {
+                    "date" : "2023-02-28",
+                    "email" : "n@n.com"
+                }
+            ]
+        }
+    ];
+    const outputDataAll = "{\"panelcontainer1675677605384\":[{\"textinput1675677621293\":\"Green\",\"numberinput1675677627103\":\"70\",\"panelcontainer1675683107466\":[{\"dropdown1675683126156\":\"yes\"},{\"dropdown1675683126156\":\"no\"},{\"dropdown1675683126156\":\"yes\"}],\"panelcontainer1675683023882\":[{\"datepicker1675683030738\":\"2023-02-24\",\"emailinput1675683037592\":\"test@test.com\"},{\"datepicker1675683030738\":\"2023-02-25\",\"emailinput1675683037592\":\"blue@blue.com\"},{\"datepicker1675683030738\":\"2023-02-14\",\"emailinput1675683037592\":\"qw@qw.com\"}]},{\"textinput1675677621293\":\"Orange\",\"numberinput1675677627103\":\"98\",\"panelcontainer1675683107466\":[{\"dropdown1675683126156\":\"no\"},{\"dropdown1675683126156\":\"yes\"},{\"dropdown1675683126156\":\"no\"}],\"panelcontainer1675683023882\":[{\"datepicker1675683030738\":\"2023-02-28\",\"emailinput1675683037592\":\"ty@ty.com\"},{\"datepicker1675683030738\":\"2023-03-04\",\"emailinput1675683037592\":\"p@p.com\"}]},{\"textinput1675677621293\":\"Horse\",\"numberinput1675677627103\":\"54\",\"panelcontainer1675683107466\":[{\"dropdown1675683126156\":\"yes\"},{\"dropdown1675683126156\":\"no\"}],\"panelcontainer1675683023882\":[{\"datepicker1675683030738\":\"2023-03-11\",\"emailinput1675683037592\":\"try@y.com\"},{\"datepicker1675683030738\":\"2023-02-28\",\"emailinput1675683037592\":\"n@n.com\"}]}]}";
```

Comment:
we should extract all json stringified data introduced in this PR to separate json files so that its easier to read and debug in future. it will also make the test file cleaner.

Reply:
This is not form json, its the submission json. Debugging is pretty simple, as you can get the Json object while debugging.
Reason I kept it here, is for keeping it in vicinity of the test (as multiple submission jsons are tested by adding and removing various instances - repeatable panel inside repeatable panel)

Reply:
debugging/making changes in stringified json is a bit difficult and error prone. Here we wouldn't want to deal with errors related to missing escape characters and quotes and thus making the life of developers complex.
whatever json it is, should be fetched from a separate json formatted file. maintaining this escaped format is difficult.

Reply:
done